test/helpers/tmp_path.rb

```diff
@@ -0,0 +1,35 @@
+module TmpPath
+  def capture_exceptions
+    super
+  rescue
+    clean_tmp_paths
+    raise
+  ensure
+    clean_tmp_paths
```

Comment:
ensure is always evaluated, so you don't need the clean_tmp_paths in `rescue`.

Reply:
Fixed